src/create_discretized_sphere.cpp

```diff
@@ -0,0 +1,109 @@
+#include<ros/ros.h>
+#include "reuleaux/PointArray.h"
+#include "geometry_msgs/Point.h"
+#include<octomap/octomap.h>
+#include<octomap/MapCollection.h>
+#include <octomap/math/Utils.h>
+#include <reuleaux/sphere_discretization.h>
+using namespace octomap;
+using namespace std;
+using namespace octomath;
+using namespace sphere_discretization;
+
+
+
+int main(int argc, char **argv)
+{
+    ros::init(argc, argv, "rand_points");
+    ros::NodeHandle n;
+  
+    ros::Publisher point_pub = n.advertise<reuleaux::PointArray>("points", 1000);
```

Comment:
Why such a large buffer?  You should only be publishing one message and so only need a buffer of size 1.  Are you streaming the points in batches or one at a time that you would need a larger buffer?